mmv1/third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -236,11 +236,11 @@ The `settings` block supports:
 
 * `collation` - (Optional) The name of server instance collation.
 
-* `disk_autoresize` - (Optional, Default: `true`) Enables auto-resizing of the storage size. Set to false if you want to set `disk_size`.
+* `disk_autoresize` - (Optional, Default: `true`) Enables auto-resizing of the storage size.
 
 * `disk_autoresize_limit` - (Optional, Default: `0`) The maximum size to which storage capacity can be automatically increased. The default value is 0, which specifies that there is no limit.
 
-* `disk_size` - (Optional, Default: `10`) The size of data disk, in GB. Size of a running instance cannot be reduced but can be increased. If you want to set this field, set `disk_autoresize` to false.
+* `disk_size` - (Optional, Default: `10`) The size of data disk, in GB. Size of a running instance cannot be reduced but can be increased.
```

Comment:
```suggestion
* `disk_size` - (Optional) The size of data disk, in GB. Size of a running instance cannot be reduced but can be increased.
```

I don't agree with including the api default here. It's not a terraform level default so I would omit it. You could mention in the description that the api sets a default value and link to corroborating api documentation.

Reply:
Should we remove the `disk_size` default note entirely? It appears to be an [undocumented default](https://github.com/GoogleCloudPlatform/magic-modules/pull/6596/files#diff-b34dfa25708e844d450cdb21f989baa23ffe466f2b817267606b10adc69048c1R307). I'll move the default into the description for other fields.

Reply:
Also, the default value for `connect_retry_interval` of 60s in our docs is mentioned as a MySQL-specific default in [PostgreSQL](https://cloud.google.com/sql/docs/postgres/admin-api/rest/v1beta4/instances#settings) and [SQL Server](https://cloud.google.com/sql/docs/sqlserver/admin-api/rest/v1beta4/instances#settings) 🤔

Reply:
> mention that it's the minimum size instead

I think sounding off that it's an API enforced minimum is a good idea !